rules/aip0134/method_rules.go

```diff
@@ -55,25 +55,31 @@ var responseMessageName = &lint.MethodRule{
 	LintMethod: func(m *desc.MethodDescriptor) []lint.Problem {
 		// Rule check: Establish that for methods such as `UpdateFoo`, the response
 		// message is `Foo` or `google.longrunning.Operation`.
-		got := m.GetOutputType().GetFullyQualifiedName()
-		want := []string{
-			m.GetName()[6:],
-			"google.longrunning.Operation",
+		want := m.GetName()[6:]
```

Comment:
Consider use `strings.Replace(m.GetName(), "Update", "", 1)`:
1. It avoids runtime errors about slice out of bound, if somehow we make a mistake in `isUpdateMethod`;
2. It is better documented -- I needed sometime to figure out what is `6` for.

Reply:
Good idea. Done.